packages/decl/test/formats/v2/normalize/unusual.test.js

```diff
@@ -56,7 +56,6 @@ describe('normalize2.unusual', () => {
 
         expect(normalize(decl).map(simplifyCell)).to.deep.equal([
             { entity: { block: 'block' }, tech: null },
-            { entity: { block: 'block', elem: 'elem1' }, tech: null },
```

Comment:
It is too complex for me. Is it okey?

Reply:
Yeah.

Read this like:
```
block: 'block',
elem: 'elem1',
mod: 'mod1',
val: 'v1',
```
and after:
```
             elems: {
                 elem: 'elem2'
             },
             mods: {
                 mod2: 'v2'
             }
```

Semantically it's the same as:
```
{
  block: 'block', elem: 'elem1', mod: 'mod1', val: 'v1',

  shouldDeps: [
    { block: 'block', elem: 'elem2' },
    { block: 'block', elem: 'elem1', mod: 'mod2', val: 'v2' },
  ]
}
```
→→→
```
block__elem1_mod1_v1
→ block__elem2
→ block__elem1_mod2_v2
```

`{ entity: { block: 'block' }, tech: null },` is redundant too.

Reply:
Guess, there is a problem with recursive `processMods({ block, mods: mod, tech });` that adding pure block entity. Probly I'm wrong.